controllers/imagecollector/imagecollector_controller.go

```diff
@@ -448,6 +448,11 @@ func (r *Reconciler) createImageJob(ctx context.Context) (ctrl.Result, error) {
 				},
 			},
 		}
+
+		log.Info("extra mount for scanner starts")
+		jobTemplate.Spec.Volumes = append(jobTemplate.Spec.Volumes, mgrCfg.ExtraScannerVolumes...)
+		scannerContainer.VolumeMounts = append(scannerContainer.VolumeMounts, mgrCfg.ExtraScannerVolumeMounts...)
+
```

Comment:
we would also need to append Volumes here. It might be easier to make the extra Volume field a part of ContainerConfig

Reply:
Do you suggest putting `Volumes` into `ManagerConfig` and `VolumeMounts` into `ContainerConfig` of `scanner`?

Reply:
We only need to add a Volumes field, since we can reference the VolumeMounts with the same name/hostPath, and can add that into ContainerConfig of scanner

Reply:
Got your point. Thank you! Code updated.